src/main/java/com/blockout/PaneParams.java

```diff
@@ -252,7 +252,7 @@ private static int parseScalableIntegerRegexMatch(Matcher m, int def, int scale)
         }
         catch (NumberFormatException | IndexOutOfBoundsException | IllegalStateException ex)
         {
-            Log.logger.warn(ex);
+            ex.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1148)